spec/integration/rails_helper_spec.rb

```diff
@@ -71,23 +62,29 @@ def tag_options(options)
     sort("other")
   end
 
+  it "should set :href on anchor" do
+    should_receive(:url_for).and_return('/example')
+    should_receive(:content_tag).with(:a, 'Field', href: '/example')
+    sort("field")
+  end
+
   it "should add no styling by default" do
     should_receive(:url_for)
-    should_receive(:a_link).with('Field', anything, hash_excluding(:class))
+    should_receive(:content_tag).with(:a, 'Field', hash_excluding(:class))
     sort("field")
   end
 
   it "should add ascending style for current ascending sort order " do
     should_receive(:url_for)
-    should_receive(:a_link).with('&#9650;&nbsp;Field', anything, hash_including(:class => 'ascending'))
+    should_receive(:content_tag).with(:a, '&#9650;&nbsp;Field', hash_including(:class => 'ascending'))
 
     params[:order] = "field ASC"
     sort("field")
   end
 
   it "should add descending style for current descending sort order " do
     should_receive(:url_for)
-    should_receive(:a_link).with('&#9660;&nbsp;Field', anything, hash_including(:class => 'descending'))
+    should_receive(:content_tag).with(:a, '&#9660;&nbsp;Field', hash_including(:class => 'descending'))
```

Comment:
Not introduced by your changes, but...

Not a big fan of these tests. Why are we using expectations, rather than asserting the HTML being generated is correct? That is a less fragile and more straightforward approach.